content/pages/sponsorship.md

```diff
@@ -0,0 +1,224 @@
+
+Since its inception, A11yTalks has hosted a variety of live talks and panels covering topics related to digital accessibility for beginners, developers, and creatives. All of these videos are streamed and then archived on the [A11yTalks YouTube channel](https://www.youtube.com/c/accessibilitytalks) with updated captions.
+
+Our content is available to the public as a free resource for digital accessibility education as well as continuing education credits for certified International Association of Accessibility Professionals (IAAP).
+
+In 2022, we assembled an [all-volunteer team](https://a11ytalks.com/pages/about), passionately contributing time and energy to sustain the project, and we became fiscally hosted by a non-profit, Open Collective Foundation (501)(c)(3).
+
+Now, we need your support to help us meet our accessibility goals, compensate our amazing speakers, and improve our live stream productions to provide the best value possible for our community.
+
+* [How to give](#how-to-give)
+* [Budget priorities](#budget-priorities)
+* [Measuring our impact](#measuring-our-impact)
+
+## How to give
+
+Your financial contributions are tax-deductible because A11yTalks is fiscally hosted by a non-profit, Open Collective Foundation 501(c)(3).
+
+### Individuals: Join the A11yCat Club
+
+You can set-up a [Monthly Recurring Donation](https://opencollective.com//a11ytalks/contribute/a11ycat-club-36522) or a [One-time Donation](https://opencollective.com//a11ytalks/contribute/one-time-donation-47987) through our Open Collective page. Each level has some contributor benefits as our way of saying “Thanks for being a part of A11yTalks!”
+
+To receive your A11yCat coaster and/or sticker, please follow the instructions on your donation confirmation email from Open Collective. These will be mailed out in monthly batches. Email us at “team [at] a11ytalks [dot] com” if you think we’ve missed you. We promise it wasn’t on purpose.
+
+<table class="centered">
+<thead class="inverse">
+<tr>
+ <th colspan="4"><a href="https://opencollective.com//a11ytalks/contribute/a11ycat-club-36522">Join the A11yCat Club</a></th>
+</tr>
+</thead>
+<tbody>
+<tr>
+  <td></td>
+  <td>$5/month<br/>($60/year, $0.16/day)</td>
+  <td>$15/month<br/>($180/year, $0.50/day)</td>
+  <td>$25/month<br/>($300/year, $0.83/day)</td>
+</tr>
+<tr>
+  <td class="left">Name listed as a donor on website</td>
+  <td>x</td>
+  <td>x</td>
+  <td>x</td>
+</tr>
+<tr>
+  <td class="left">Name on slide at the beginning of<br/>each A11yTalk</td>
+  <td>x</td>
+  <td>x</td>
+  <td>x</td>
+</tr>
+<tr>
+  <td class="left">A11yTalks sticker</td>
+  <td>1</td>
+  <td>2</td>
+  <td>3</td>
+</tr>
+<tr>
+  <td class="left">A11yTalks coaster</td>
+  <td>x</td>
+  <td>x</td>
+  <td></td>
+</tr>
+</tbody>
+</table>
+
+<table class="centered">
+<thead class="inverse">
+<tr>
+ <th colspan="4"><a href="https://opencollective.com//a11ytalks/contribute/one-time-donation-47987">Give a One-Time Donation</a></th>
```

Comment:
Same as the other: We can remove the black background. Can we make the link a button or just an h4 linked?